bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -60,32 +60,33 @@
 neuron_stats_columns = [
     ['UID', 'uid', '{:.0f}', 'cyan'],  # neuron UID
```

Comment:
Maybe we should put this on the docs haha

Reply:
Yes, we'll do that once the table columns have been finalized.